xc7/utils/prjxray_form_channels.py

```diff
@@ -202,82 +209,134 @@ def import_grid(db, grid, conn):
         # tile: pkey name tile_type_pkey grid_x grid_y
         c.execute(
             """
-INSERT INTO tile(name, tile_type_pkey, grid_x, grid_y)
+INSERT INTO phy_tile(name, tile_type_pkey, grid_x, grid_y)
 VALUES
   (?, ?, ?, ?)""",
             (tile, tile_types[gridinfo.tile_type], loc.grid_x, loc.grid_y)
         )
 
-    build_other_indicies(c)
     c.connection.commit()
 
+    build_phy_tile_grid_indicies(c)
+
 
-def import_nodes(db, grid, conn):
+def import_nodes(db, conn, tile_types_to_split, tile_wire_name_map):
```

Comment:
This code doesn't need to change at all?  It should only operate on phy_tiles, and never care about VPR logical tiles.

Reply:
I think it has to. Because the `wire` table is filled there. It contains wire to tile instance relation. It is later used when tracks are formed and tile_pkeys must point to VPR tiles.